strongbox-resources/strongbox-storage-api-resources/src/main/resources/etc/conf/strongbox.yaml

```diff
@@ -51,6 +51,7 @@ configuration:
           type: hosted
           allowsForceDeletion: true
           checksumHeadersEnabled: true
+          strictChecksum: true
```

Comment:
I think this should be called either `strictChecksumValidation`, or at the very least `strictChecksums`.

Reply:
Done!